test/acceptance/specs/content-page.js

```diff
@@ -0,0 +1,11 @@
+const ContentPage = require('../pageobjects/content-ache.page');
+
+describe('stomach ache page', () => {
+  it('should contain a title', () => {
+    ContentPage.open();
+
+    ContentPage.header.isVisible().should.be.true;
+    ContentPage.emergencyBox.isVisible().should.be.true;
```

Comment:
I think `emergencyBox` and `gpBox` are going to be content dependent. They're not always going to be present.